sdk-base/src/main/java/com/mapbox/maps/plugin/annotation/ClusterOptions.kt

```diff
@@ -0,0 +1,70 @@
+package com.mapbox.maps.plugin.annotation
+
+import android.graphics.Color
+
+/**
+ * Options to show and configure symbol clustering with using SymbolManager.
+ * <p>
+ * It exposes a minimal of configuration options, a more advanced setup can be created manually with
+ * using CircleLayer and SymbolLayers directly.
+ * </p>
+ */
+data class ClusterOptions(
+
+  /**
+   * If the data is a collection of point features, setting this to true clusters the points
+   * by radius into groups. Cluster groups become new `Point` features in the source with additional properties:
+   * - `cluster` Is `true` if the point is a cluster
+   * - `cluster_id` A unqiue id for the cluster to be used in conjunction with the
+   * [cluster inspection methods](https://www.mapbox.com/mapbox-gl-js/api/#geojsonsource#getclusterexpansionzoom)
+   * - `point_count` Number of original points grouped into this cluster
+   * - `point_count_abbreviated` An abbreviated point count
+   */
+  val cluster: Boolean = true,
+
+  /**
+   * Radius of each cluster if clustering is enabled. A value of 512 indicates a radius equal
+   * to the width of a tile, 50 by default.
+   */
+  val clusterRadius: Long = 50,
```

Comment:
Can we at least move all numeric constants to some file?

Reply:
IMHO, it is more clear for users to get the default values here.